### DISCUSSION TITLE
代码在运行时如何获取antd的版本号？

### DISCUSSION THREAD

**Discussion by yuqiang1999** [ant-design/ant-design#41748] [Q&A] [answered]
场景：我提供一个组件的npm包，该组件需要做 antd4、antd5的属性兼容，需要获取项目里依赖的antd版本
尝试使用 import {version} from 'antd';  项目里使用 4.x 时可行，
5.x 版本会报错：Uncaught Error: Module "./antd/es/version/style" does not exist in container. 
有什么办法获取版本号吗

**Comment by afc163** [comment, marked as answer]
你需要把 babel-plugin-import 关掉。
